tests/phpunit/elementor/core/app/import-export/test-import.php

```diff
@@ -0,0 +1,479 @@
+			foreach ( $post_type_taxonomies as $taxonomy_key => $imported_terms ) {
+				foreach ( $imported_terms as $imported_term ) {
+					$imported_term_id = reset( $imported_term['id'] );
+					$imported_term_slug = reset( $imported_term['slug'] );
+
+					$term = get_term( $imported_term_id );
+					$this->assertTrue( $term->taxonomy === $taxonomy_key );
+					$this->assertTrue( $term->slug === $imported_term_slug );
+				}
+			}
+		}
+	}
+
+	private function recursive_unset( &$elements, $unwanted_key ) {
+		foreach ( $elements as &$element ) {
+			unset( $element[ $unwanted_key ] );
+			$this->recursive_unset( $element['elements'], $unwanted_key );
+		}
+	}
+
+	// Assertions for elementor content by testing if the post containing the "JSON" content.
+	// The "JSON" is located in the content folder inside the kit.
+	private function assert_valid_elementor_content( $result, $manifest, $zip_path) {
+		$import_process_tmp_dir = Plugin::instance()->uploads_manager->extract_and_validate_zip( $zip_path )['extraction_directory'];
+		
+		foreach ( $manifest['content'] as $elementor_post_type => $elementor_posts ) {
+			foreach ($elementor_posts as $post_id => $post_settings) {
+				$expected_post_data = ImportExportUtils::read_json_file($import_process_tmp_dir . '/content/' . $elementor_post_type . '/' . $post_id);
+				$expected_post_content = $expected_post_data['content'];
+
+				$imported_post_id = $result['content'][$elementor_post_type]['succeed'][$post_id];
+				$new_post = Plugin::$instance->documents->get($imported_post_id, false);
+				$post_content = $new_post->get_json_meta('_elementor_data');
+
+				$this->recursive_unset($post_content, 'isInner');
+				$this->recursive_unset($expected_post_content, 'isInner');
+
+				$this->assertEquals($expected_post_content, $post_content);
+			}
+		}
+	}
+
+	// Assertions for imported taxonomies and imported Elementor content by testing the terms of every Elementor post.
+	private function assert_valid_terms_with_elementor_content( $result, $manifest) {
+		foreach ( $manifest['content'] as $elementor_post_type => $elementor_posts ) {
+			foreach ( $elementor_posts as $post_id => $post_settings ) {
+				$expected_post_terms = $manifest['content'][ $elementor_post_type ][ $post_id ]['terms'];
+				$imported_post_id = $result['content'][ $elementor_post_type ]['succeed'][ $post_id ];
+
+				foreach ( $expected_post_terms as $term ) {
+					$post_terms = get_the_terms( $imported_post_id, $term['taxonomy'] );
+					$this->assertEquals( $term['slug'], $post_terms[0]->slug );
+				}
+			}
+		}
+	}
+
+	// Assertions for imported taxonomies and imported WP content by testing the terms of every WP post.
+	private function assert_valid_terms_with_wp_content( $result ) {
+		foreach ( $result['wp-content'] as $wp_post_type => $wp_posts ) {
+			foreach ( $wp_posts['succeed'] as $new_post_id ) {
+				if ( empty( $result['taxonomies'][ $wp_post_type ] ) ) {
+					continue;
+				}
+
```

Comment:
if it's always an array you can skip this condition

Reply:
Done